core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/dot-edit-content-category-field.component.ts

```diff
@@ -115,8 +120,20 @@ export class DotEditContentCategoryFieldComponent implements OnInit {
         this.#componentRef.instance.closedSidebar
             .pipe(takeUntilDestroyed(this.#destroyRef), delay(CLOSE_SIDEBAR_CSS_DELAY_MS))
             .subscribe(() => {
+                this.updateCategoryFieldControl();
+                // enable the show sidebar button
                 this.disableSelectCategoriesButton.set(false);
-                this.sidebarHost.viewContainerRef.clear();
+                this.removeDotCategoryFieldSidebarComponent();
             });
     }
+
+    private updateCategoryFieldControl(): void {
```

Comment:
@oidacra In the other components, this approach is slightly different:

```ts
get formControl(): FormControl {
    return this.#controlContainer.control.get(this.field.variable) as FormControl<string>;
}

private updateCategoryFieldControl(): void {
   this.#formControl.setValue(this.store.selectedCategoriesValues());
}
```

Your way works perfectly, but I think we should use one approach in all components.

Reply:
Yes, Changed to use get.